modules/EvseManager/powermeter/powermeterImpl.hpp

```diff
@@ -0,0 +1,63 @@
+#ifndef POWERMETER_POWERMETER_IMPL_HPP
```

Comment:
Again, no license header

Reply:
YetiDriver and EvseManager updated with edm mod update --force to fix license tags